fun-facts.css

```diff
@@ -0,0 +1,91 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #785F37;
+}
+
+section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    width: 100%;
+}
+
+
+ul {
+    float: right;
+    list-style-type: none;
+    margin-top: 25px;
+    margin-left: -60px;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    padding: 14px 16px;
+}
+
+ul li{
+    display: inline-block;
+    float: right;
+}
+
+ul li a{
+    text-decoration: none;
+    color: #fff;
+    padding: 14px 20px;
+    border: 1px solid transparent;
+    transition: 0.6s ease;
+    
+}
+
+ul li a:hover{
+    background-color: #fff;
+    color: #000;
+}
+
+ul li.active a {
+    background-color: #fff;
+    color: #000;
+}
+
+.logo {
+    float: left;
+    width: 6%;
+    height: auto;
+    margin-top: 2%;
+}
+
+.main {
+    max-width: 1200px;
+    margin: auto;
+    overflow-y: scroll;
```

Comment:
I would suggest removing this line as it creates a scrollbar on the navigation bar.

```suggestion

```